ansible/README.md

```diff
@@ -1,76 +0,0 @@
-## Ansible Provisioner for Vitess
-
-### Testing individual roles
-
-Most roles in the playbook have been configured to be tested with molecule. These can be run by issuing
```

Comment:
Do we want to keep the readme and update it instead of removing it entirely?

Reply:
thank you, it should not be removed entirely, I am still due for a small update on the old files/tests